docs/other-topics/transactions.md

```diff
@@ -4,7 +4,7 @@ Sequelize does not use [transactions](https://en.wikipedia.org/wiki/Database_tra
 
 Sequelize supports two ways of using transactions:
 
-1. **Unmanaged transactions:** Committing and rolling back the transaction should be done manually by the user (by calling the appropriate Sequelize methods).
+1. **Unmanaged transactions:** Committing and rolling back the transaction should be done manually by the user (by calling the appropriate Sequelize methods). Except for deadlocks, it will does automatically rollback, read [#12841](https://github.com/sequelize/sequelize/pull/12841).
```

Comment:
Reworded a bit because that is only true for mariadb/mysql

```suggestion
1. **Unmanaged transactions:** Committing and rolling back the transaction should be done manually by the user (by calling the appropriate Sequelize methods).  
  Note that innoDB (MariaDB and MySQL) will still automatically rollback transactions in case of deadlock. [Read more on this here](https://github.com/sequelize/sequelize/pull/12841).
```